### PR TITLE
Misc post carbon tagging

### PR DESCRIPTION
**Pull request by Dieterbe**
feel free to be as harsh on this PR as i've been on the tagging PR ;-)

### CODE REVIEW

File: carbon/schemas.go [hunk 1]
```diff
@@ -36,18 +37,18 @@ func parseMetric(buf []byte, schemas *conf.Schemas, orgId int) (*schema.MetricDa
 	tags := metric[1:]
 	for _, v := range tags {
 		if v == "" || !strings.Contains(v, "=") || v[0] == '=' {
-			return nil, fmt.Errorf(errFmt, msg, "unable to parse tag")
+			return nil, errBadTag
```
Comment (DanCech) [inline carbon/schemas.go]: Might be good here to tell the user which tag segment was bad

Comment (jtlisi) [reply carbon/schemas.go]: Based on what you said in the previous PR @Dieterbe, would a log statement here with the tag work?

Comment (Dieterbe) [reply carbon/schemas.go]: the caller of parseMetric can (and does) log the entire metric key that was given to parseMetric, when there was an error. I think that's the best trade-off

Comment (jtlisi) [reply carbon/schemas.go]: Ahh, that seems best

**Pull request merged**